gridstatus/spp.py

```diff
@@ -704,6 +704,55 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    @support_date_range("5_MIN")
+    def get_day_ahead_marginal_clearing_prices(self, date, end=None, verbose=False):
+        """Get Day Ahead Marginal Clearing Prices. Updated every 5 minutes.
+        Available at https://portal.spp.org/pages/da-mcp#
+
+        Args:
+            date: date to get data for
+            end: end date
+            verbose: print url
+
+        Returns:
+            pd.DataFrame: Day Ahead Marginal Clearing Prices
+        """
+        if date == "latest":
+            date = pd.Timestamp.now(tz=self.default_timezone)
```

Comment:
There isn't a latest file as for operating prices so this retrieves the file for the current time.

Reply:
i'd say no need to support "latest" for this since it it is daily ahead data. if the user wants latest day they can use "today". raise error if latest is provided and say to use "today" or specify day